server/apps/rules/routing_rules.py

```diff
@@ -21,6 +21,15 @@
 STATE_ROUTED = 'routed'
 
 
+WEEKDAYS = ['MON', 'TUE', 'WED', 'THU', 'FRI', 'SAT', 'SUN']
```

Comment:
Would `Enum` be a more suitable data type here? If we decide for it, we can use it even in Python3.3, there exists a backport (called `enum34`).